stat/distuv/logistic_test.go

```diff
@@ -0,0 +1,115 @@
+package distuv
+
+import (
+	"math"
+	"testing"
+)
+
+func TestLogisticParameters(t *testing.T) {
+	l := Logistic{Mu: 0, S: 0}
+
+	if l.NumParameters() != 2 {
+		t.Fail()
```

Comment:
Please use `t.Errorf(...)`, like for example in https://github.com/gonum/gonum/blob/master/stat/distuv/beta_test.go